src/main/resources/standardTestCases/invalid_wdl.test

```diff
@@ -0,0 +1,10 @@
+name: invalid_wdl
+testFormat: workflowfailure
+
+files {
+  wdl: failures/invalid_wdl/invalid_wdl.wdl
+}
+
+metadata {
+  "failures.0": "Unable to load namespace from workflow: ERROR: Unexpected symbol (line 1, col 15) when parsing 'task'.\n\nExpected lbrace, got <string:1:15 rbrace \"fQ==\">.\n\ntask blahblah }{\n              ^\n\n$task = :task :identifier :lbrace $_gen3 $_gen4 :rbrace -> Task( name=$1, declarations=$3, sections=$4 )\n     "
```

Comment:
It will, but that's the message we get...

On the plus side, wdl4s doesn't change too often, and it's easy enough to update?